docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-space-manager.md

```diff
@@ -0,0 +1,42 @@
+<!--example1-space-manager-start-->
+
+### Create Kind cluster for space management
+
+```shell
+kind create cluster --name sm-mgt
+KUBECONFIG=~/.kube/config kubectl config rename-context kind-sm-mgt sm-mgt
+export SM_CONFIG=~/.kube/config
+export SM_CONTEXT=sm-mgt
+```
+
+### The space-manager controller
+
+You can get the latest version from GitHub with the following command,
+which will get you the default branch (which is named "main"); add `-b
+$branch` to the `git` command in order to get a different branch.
+
+```{.bash}
+git clone {{ config.repo_url }}
+cd kubestellar
+```
+
+Use the following commands to build and add the executables to your
+`$PATH`.
+
+```shell
+cd space-framework
+make build
+export PATH=$(pwd)/bin:$PATH
+```
+Next deploy the space framework CRDs in the space management cluster.
+```shell
+KUBECONFIG=$SM_CONFIG kubectl --context sm-mgt apply -f config/crds/
+cd ..
+```
+Finally, start the space-manager controller.
+
+```shell
+space-manager --kubeconfig $SM_CONFIG --context sm-mgt -v 2 &
```

Comment:
What happened to switching to `-v=4` ?

Reply:
You mean we decided that kubestellar will be using -v=4 henceforth?  I just searched the code and we always use 2.

Reply:
Yes, we discussed this in a meeting and decided that `-v=2` is for mere users while `-v=4` is for contributors. The example1 and user-quickstart-test are for contributors.